### PR TITLE
Feature/radio synchronize

### LINKED ISSUE #108: Syncronize RadioButton element between Java and C#
Syncronize RadioButton element between Java and C#
Use Java version as a reference
List of cheks:
1. Element should have the same number of properties and methods
2. Names of properties and methods should be the same (with paying respect to language-specific, off course )
3. Element should be covered with the same amount of tests with the same functionality and checks
4. Tests should be located in the directory with the same name and same hierarchy
Documentation for the element should be corrected accordingly

### PR DESCRIPTION
**Pull request by DmitrievDmitrii**
fixed radio buttons according to java

`closes #108 `

### CODE REVIEW

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    public class RadioButtons : UIElement, IRadioButtons
+    {
+        public By RadioButtonLocator { get; set; }
+
+        public By LabelLocator { get; set; }
+
+        private List<Label> Labels => FindElements(LabelLocator)
+            .Select(element => UIElementFactory.CreateInstance<Label>(LabelLocator, this, element)).ToList();
+
+        private List<UIElement> Radios => FindElements(RadioButtonLocator)
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: we can use RadioButton class here

File: JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public interface IRadioButtons : IBaseUIElement, IGetValue<string>
+    {
+        By RadioButtonLocator { get; set; }
+        By LabelLocator { get; set; }
+        void Select(string value);
+        void Select(int index);
+        new string Selected();
+        List<string> Values();
+    }
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs]: we can also add something like SelectAll() and UnselectAll()

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        }
+
+        public List<string> Values()
+        {
+            return Labels.Select(label => label.Text.Trim()).ToList();
+        }
+
+        public string Value => Selected();
+
+        private static bool IsChecked(UIElement radioButton) => radioButton.GetAttribute("checked") != null;
```
Comment (elv1s42) [inline JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: Please do not use a static methods.
Also we don't need this method at all, looks like it is used only once.

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs (hunk 1 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: I can't find it in solution. Is it implemented in some branch?

File: JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs (hunk 2 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs]: Not sure if this is suitable for radio buttons.

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs (hunk 3 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: method removed

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs (hunk 1 above)
Comment (elv1s42) [reply JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: @DmitrievDmitrii , please find it here:
https://github.com/jdi-testing/jdi-light-csharp/blob/master/JDI.Light/JDI.Light/Elements/Common/RadioButton.cs

File: JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs (hunk 2 above)
Comment (elv1s42) [reply JDI.Light/JDI.Light/Interfaces/Complex/IRadioButtons.cs]: Agreed.

File: JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs (hunk 1 above)
Comment (DmitrievDmitrii) [reply JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs]: this is previous name of this class.
i renamed it and moved to complex namespace, according to java

**Pull request merged**